uvicorn/supervisors/multiprocess.py

```diff
@@ -90,9 +90,13 @@ def kill(self) -> None:
         # In Unix, the method will send SIGKILL to the process.
         self.process.kill()
 
-    def join(self) -> None:
+    def join(self, join_timeout: float | None = None) -> None:
```

Comment:
Can we call it only `timeout`?

```suggestion
    def join(self, timeout: float | None = None) -> None:
```